Source/Scene/ModelExperimental/ModelRenderResources.js

```diff
@@ -67,5 +71,14 @@ export default function ModelRenderResources(model) {
    *
    * @private
    */
-  this.renderStateOptions = {};
+  this.renderStateOptions = clone(
+    RenderState.fromCache({
+      depthTest: {
+        enabled: true,
+        func: DepthFunction.LESS_OR_EQUAL,
+      },
+      blending: BlendingState.DISABLED,
+    }),
+    true
+  );
```

Comment:
Moved the default render state from `PrimitiveRenderResources` to `ModelRenderResources` so that it starts at the top and propagates down.